### DISCUSSION TITLE
Beta version is not allowed

### DISCUSSION THREAD

**Discussion by PakhomovAlexander** [Homebrew/discussions#3959] [Writing Formulae/Casks] [answered]
Hello, I am writing a formula and get the following audit error:

`Stable version URLs should not contain beta`

Is there any option to skip this particular audit check? The software I am going to add to homebrew is an OS Apache project but it is in beta now. Shall I wait for the stable version? Thanks.

**Comment by SMillerDev** [comment, marked as answer]
> Shall I wait for the stable version? Thanks.

Yes: https://docs.brew.sh/Acceptable-Formulae#stable-versions
